README.md

```diff
@@ -23,10 +23,21 @@ For more information, see:
 
 This project exists thanks to all the people who contribute and who have contributed in the past, whether as part of the long history of thousands of contributions to WordPress from many different people, or as contributions to ClassicPress itself.
 
-You can see a list of WordPress contributors by going to the [WordPress releases page](https://wordpress.org/news/category/releases/) and looking at the credits for each release.  All changes in WordPress 4.9.8 and earlier are part of ClassicPress, along with selected changes from later releases.
+You can help Fix bugs by submitting Pull Requests. Most of the bugs are shared via the [issues page](https://github.com/ClassicPress/ClassicPress/issues). Here are some pointers on how to:
```

Comment:
"You can help Fix bugs" &rarr; "You can help fix bugs"

Reply:
I see some content ~~removed~~ Was this intentional?

Reply:
What text was removed? I haven't made any changes here, and I'm not sure what you're referring to

Reply:
My Git was showing me something else. Do we need to showcase this line?

> You can see a list of WordPress contributors by going to the [WordPress releases page](https://wordpress.org/news/category/releases/) and looking at the credits for each release.  All changes in WordPress 4.9.8 and earlier are part of ClassicPress, along with selected changes from later releases.